config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-3-upgrades.yaml

```
@@ -329,7 +329,54 @@ periodics:
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "4"
 
-- name: periodic-cluster-api-e2e-workload-upgrade-1-25-latest-release-1-3
+- name: periodic-cluster-api-e2e-workload-upgrade-1-25-1-26-release-1-3
+  interval: 24h
+  decorate: true
+  decoration_config:
+    gcs_credentials_secret: "" # Use workload identity for uploading artifacts
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api
+      base_ref: release-1.3
+      path_alias: sigs.k8s.io/cluster-api
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+  spec:
+    serviceAccountName: prowjob-default-sa
+    containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221223-736a4da5ba-1.25
```

```suggestion
      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221221-461b6105cf-1.25
```

Please change the image and then rebase the entire PR to latest master + ensure that the new jobs are then using the same images as the other jobs in the same file. Thx!